src/main/java/com/swp/user/controller/RelationshipController.java

```diff
@@ -0,0 +1,30 @@
+package com.swp.user.controller;
+
+import com.swp.user.domain.RelationshipService;
+import com.swp.user.dto.RelationshipRequestDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+@RequiredArgsConstructor
+@Controller
+public class RelationshipController {
+
+    private final RelationshipService relationshipService;
+
+    @PostMapping(value = "/v1/relationship")
+    public ResponseEntity<?> followUser(@RequestBody RelationshipRequestDto relationshipRequestDto){
+        relationshipService.createRelationship(relationshipRequestDto);
+        return new ResponseEntity<> (relationshipRequestDto,HttpStatus.OK);
+    }
```

Comment:
```suggestion
    @PostMapping(value = "/v1/relationship")
    public RelationshipResponseDto followUser(@RequestBody RelationshipRequestDto relationshipRequestDto){
        return relationshipService.createRelationship(relationshipRequestDto);
    }
```

`@RestController`를 적용하면 이렇게 바뀔 수 있겠네요!

Reply:
>1번은 확인했습니다. 2번은 테스트 할 때 확인하려고 해 놓았는데 의미가 없으니 void로 바꾸고, 3번은 구현하다 쓸 수도 있을 것 같아서 추가했는데, 아직까지는 필요가 없네요 수정하겠습니다!